client/src/components/Form.js

```diff
@@ -13,6 +13,7 @@ export default function Form({ submitFunction }) {
   };
 
   const [member, setMember] = useState(initialMember);
+  console.log('member', member);
```

Comment:
I think this should be deleted before merging :)

Reply:
Ohh yes, forgot about this one!